source/guides/network/supernode-enrollment.rst

```diff
@@ -141,30 +183,32 @@ After following that guide, you need to **install a monitoring agent** and **ann
 
       ./agent-linux.bin --config agent.properties
 
-5. **Send the enrolling message**
+   .. note:: Once the agent starts running, it will poll the Controller every 2 minutes until the enrollment is completed and new communication certificates are generated. At that point the agent just listens for commands from the Controller.
+
+5. **Send the enrollment message**
 
    The last bit is to **notify the Controller** that your node wants to enroll in the Supernode program. This is done through a conventional :ref:`Transfer Transaction <transfer-transaction>` with no mosaics and a special message:
 
    .. code-block:: text
 
-      enrol NODE_PUBLIC_KEY AGENT_URL
+      enroll AGENT_URL BASE64_ENCODED_AGENT_CA_CSR
 
-   - Replace ``NODE_PUBLIC_KEY`` with your node's **transport** public key. You can get it from the ``nodePublicKey`` field in http://localhost:3000/node/info, for example, when your node is running.
+   - Replace ``AGENT_URL`` with ``https://`` + the host where you are running the agent + ``:7881``. This URL must be **publicly accessible**. For example: `https://my-symbol-node.com:7881 <https://my-symbol-node.com:7881>`__. IP addresses are also valid. Use the port number you specified in step 3 above if you didn’t use the standard one.
 
-   - Replace ``AGENT_URL`` with ``https://`` + the host where you are running the agent + ``:7880``. This URL must be **publicly accessible**. For example: ``https://my-symbol-node.com:7880``. IP addresses are also valid. Use the port number you specified in step 3 above if you didn't use the standard one.
+   - Replace ``BASE64_ENCODED_AGENT_CA_CSR`` with the output of step 2.
 
    Finally, the recipient address for this transaction is:
 
    - ``NDG2F6IHON7EDOXZCHSTSJ2YMUHDFXAQ2EUZHFA`` for MAINNET.
    - ``TDL73SDUMPDK7EOF7H3O4F5WB5WHG2SX7XUSFZQ`` for TESTNET.
 
-   The transaction can be announced using :doc:`symbol-cli <../../cli>`:
+   The transaction can then be announced using :doc:`symbol-cli <../../cli>`:
 
    .. code-block:: symbol-cli
 
       symbol-cli transaction transfer --mode normal --sync \
                  --recipient-address NDG2F6IHON7EDOXZCHSTSJ2YMUHDFXAQ2EUZHFA \
-                 --message "enrol NODE_PUBLIC_KEY AGENT_URL" \
+                 --message "enrol AGENT_URL BASE64_ENCODED_AGENT_CA_CSR" \
                  --mosaics @symbol.xym::0
 
    **This transaction must be signed by your node's main account**, so make sure it is the default profile in ``symbol-cli``.
```

Comment:
Should we mention the `-max-fee` parameter here? (that they can set a higher max fee in case it doesn't get confirmed.)

Reply:
Yeah, good idea.